create-standard-riglet.sh

```diff
@@ -10,12 +10,13 @@ read -p "Are you sure you want to proceed?  " -n 1 -r
 echo
 if [[ $REPLY =~ ^[Yy]$ ]]
 then
+  make create-deps && \
   echo "Y" | make create-foundation ENV=integration && \
   echo "Y" | make create-compute ENV=integration && \
   echo "Y" | make create-foundation ENV=staging && \
   echo "Y" | make create-compute ENV=staging && \
   echo "Y" | make create-foundation ENV=production && \
   echo "Y" | make create-compute ENV=production && \
-  echo "Y" | make create-build REPO=bookit-api REPO_BRANCH=master CONTAINER_PORT=8080 LISTENER_RULE_PRIORITY=100 && \
```

Comment:
I couldn't really follow where the branch is specified going forward.

Reply:
So, the branch still defaults to `master` (line 10 of Makefile).  Can override that in the same way.